converter/src/main/java/gov/cms/qpp/conversion/model/error/Detail.java

```diff
@@ -106,6 +107,10 @@ private static String computeLocation(Node node) {
 			if (!StringUtils.isEmpty(possibleMeasureId)) {
 				location.append(" ");
 				location.append(possibleMeasureId);
+				String possibleElectronicMeasureId = MeasureConfigHelper.getMeasureConfigIdByUuidOrDefault(possibleMeasureId);
+				if (!StringUtils.isEmpty(possibleElectronicMeasureId)) {
```

Comment:
Since you’re appending to a string builder, don’t use string concat, instead just call append 3 times

Reply:
Good catch!